ui/prometheus-plugin/src/plugins/prometheus-time-series-query/get-time-series-data.ts

```diff
@@ -87,6 +87,7 @@ export const getTimeSeriesData: TimeSeriesQueryPlugin<PrometheusTimeSeriesQueryS
         name,
         values: values.map(parseValueTuple),
         formattedName,
+        labels: metric,
```

Comment:
isn't `metric` just the flat string of the labels? or is it already transformed to a JSON here?

Reply:
It’ll be already transformed to JSON!